manual/create-new-environment.md

```diff
@@ -0,0 +1,58 @@
+# Create a new environment
+This is a guide to creating a new tdr environment in a clean AWS account.
```

Comment:
If we were deploying a completely new environment (i.e. one with a different name to the three we have now), presumably we'd have to update quite a lot of the Terraform scripts? Might be worth just stating that this guide assumes that you're deploying intg, staging or prod to a new account, since they're already configured in the Terraform.

Reply:
Yep, fair point. I've put in a sentence to say that.